src/SFA.DAS.QnA.Database/SFA.DAS.QnA.Database.sqlproj

```diff
@@ -8,7 +8,7 @@
     <SchemaVersion>2.0</SchemaVersion>
     <ProjectVersion>4.1</ProjectVersion>
     <ProjectGuid>{821f1855-08db-4c78-a4b9-d526c55a8636}</ProjectGuid>
-    <DSP>Microsoft.Data.Tools.Schema.Sql.SqlAzureV12DatabaseSchemaProvider</DSP>
+    <DSP>Microsoft.Data.Tools.Schema.Sql.Sql150DatabaseSchemaProvider</DSP>

```

Comment:
I don't know if this is a significant change, should be be reverted?

Reply:
Reverted